File: session/opencv/p1/main.py
import cv2

# Cargar imagen
img = cv2.imread("test.jpg")

# Verificar si se cargó correctamente
if img is None:
    print("Image not found!")
else:
    # Mostrar imagen
    cv2.imshow("Display", img)
    cv2.waitKey(0)
    cv2.destroyAllWindows()

File: session/opencv/p1/main.cpp
#include <opencv2/opencv.hpp>
#include <iostream>

using namespace cv;

int main() {
    Mat img = imread("test.jpg");
    if (img.empty()) {
        std::cout << "Image not found!" << std::endl;
        return -1;
    }

    imshow("Display", img);
    waitKey(0);

    return 0;
}
